src/Aliases.jl

```diff
@@ -8,6 +8,9 @@ include(joinpath(pathof(AbstractAlgebra), "..", "Aliases.jl"))
 @alias is_less isless
 @alias is_real isreal
 
+@alias eigenvalues eigvals
+@alias eigenvalues_simple eigvals_simple
```

Comment:
What does this function even do? I am asking as we didn't talk about it today.

Reply:
According to the doc string, it computes eigenvalues, but the function only works if the matrix has only "simple eigenvalues" (I assume simple = multiplicity 1).